pkg/tfvars/baremetal/baremetal.go

```diff
@@ -115,41 +118,28 @@ func TFVars(libvirtURI, apiVIP, imageCacheIP, bootstrapOSImage, externalBridge,
 			rootDevice["name"] = profile.RootDeviceHints.DeviceName
 		}
 
-		// Instance Info
-		// The machine-os-downloader container downloads the image, compresses it to speed up deployments
-		// and then makes it available on bootstrapProvisioningIP via http
-		// The image is now formatted with a query string containing the sha256sum, we strip that here
-		// and it will be consumed for validation in https://github.com/openshift/ironic-rhcos-downloader
-		imageURL, err := url.Parse(image)
-		if err != nil {
-			return nil, err
-		}
-		imageURL.RawQuery = ""
-		imageURL.Fragment = ""
-		// We strip any .gz/.xz suffix because ironic-machine-os-downloader unzips the image
-		// ref https://github.com/openshift/ironic-rhcos-downloader/pull/12
-		imageFilename := path.Base(strings.TrimSuffix(imageURL.String(), ".gz"))
-		imageFilename = strings.TrimSuffix(imageFilename, ".xz")
-		cachedImageFilename := "cached-" + imageFilename
-		cacheImageURL := fmt.Sprintf("http://%s/images/%s/%s", net.JoinHostPort(imageCacheIP, "80"), imageFilename, cachedImageFilename)
-		cacheChecksumURL := fmt.Sprintf("%s.md5sum", cacheImageURL)
-		instanceInfo := map[string]interface{}{
-			"image_source":   cacheImageURL,
-			"image_checksum": cacheChecksumURL,
-		}
-
 		// This is the only place where we need to set instance_info capabilities,
 		// if we need to add another capabilitie we need merge the values
 		// and ensure they are in the `key1:value1,key2:value2` format
+		instanceInfo := make(map[string]interface{})
 		if host.BootMode == baremetal.UEFISecureBoot {
 			instanceInfo["capabilities"] = "secure_boot:true"
 		}
 
+		// deploy_steps is set when a custom deployment is desired. We will use ironic's custom deployment
+		// interface to use live ISO based installer. Currently this value is static but may be configurable
+		// in the future.
+		var hostDeploySteps string
+		if strings.Contains(accessDetails.BootInterface(), "redfish-virtual-media") {
```

Comment:
Should be removed, has nothing to do with virtual media.

Reply:
Right. You did clarify this for me a while ago. This is an old vestigial condition. I'll remove it.